usr/share/rear/layout/prepare/GNU/Linux/110_include_lvm_code.sh

```diff
@@ -68,9 +68,9 @@ create_lvmgrp() {
     local vg=${vgrp#/dev/}
 
     cat >> "$LAYOUT_CODE" <<EOF
-create_volume_group=1
-create_logical_volumes=1
-create_thin_volumes_only=0
+create_volume_group+=( "$vg" )
```

Comment:
could you provide an explanatory comment directly in the code
that describes how that code works now - e.g. something like
a summary of your explanation you gave in
https://github.com/rear/rear/pull/2691#issue-1015457982

In particular because I am not a LVM expert
I would likely not sufficiently understand how that works
so if I have to fix something therein at some later time
I may too easily break something by accident.